src/main/java/veryfi/Constants.java

```diff
@@ -21,7 +21,7 @@ private Constants() {
     /**
      * header for HttpRequest
      */
-    public static final String USER_AGENT_JAVA = "Java Veryfi-Java/1.0.11";
+    public static final String USER_AGENT_JAVA = "Java Veryfi-Java/1.0.12";
```

Comment:
too many changes for a minor version -
if something is breaking, should be 2.0.0, otherwise 1.1.0

Reply:
Changed to 2.0.0 (added missing parameters to `getDocuments()` )
✅